funnel/models/commentvote.py

```diff
@@ -230,6 +232,18 @@ def title(self):
         else:
             return _("{user} commented").format(user=self.user.pickername)
 
+    @property
+    def badge(self):
```

Comment:
Should be `badges` returning a set. User can be both `crew` and `proposer` for instance.